cli/flox-rust-sdk/src/models/environment/mod.rs

```diff
@@ -317,7 +317,7 @@ pub enum EnvironmentError2 {
     // todo: candidate for impl specific error
     // * only path and managed env are defined in .Flox
     // region: path env open
-    #[error(".flox directory not found")]
+    #[error("Did not find an environment in the current directory.")]
```

Comment:
nit: Current directory is to specific
lets leave it either generic, or include the actual path that was looked up

Reply:
Also having _some_ errors be sentences with periods and others not is weird.
The lowercase ones were meant for compatibility with error chains (`error1: error2: ...`) Using sentences and a less compact error chain might be more compatible eventually.

Reply:
Yeah I botched the `-d` case

Reply:
Fixed